src/scss/mixins/visibility.scss

```diff
@@ -0,0 +1,16 @@
+@mixin sr-only {
+	border: 0;
+	clip: rect(0, 0, 0, 0);
+	height: 1px;
+	margin: -1px;
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	text-indent: 200%;
+	white-space: nowrap;
+	width: 1px;
+}
+
+@mixin hidden {
+	display: none !important;
```

Comment:
Is the `!important` necessary?

Reply:
Yeah, this is to prevent being overridden by more specific selectors or the order of styles.